### DISCUSSION TITLE
setting funny geometries

### DISCUSSION THREAD

**Discussion by phscald** [lululxvi/deepxde#1303] [Q&A]
I am trying to implement a complex geometry in my problem, but couldn't find the solution with deepxde so far.
I've got the domain defined in the Figure below. The domain does not include the region inside the blue blobs. 
Is there a easy way to upload these points and remove their shapes from a main geometry.Rectangle?

![image](https://user-images.githubusercontent.com/128174498/236962346-cd6fc4f8-0710-492d-b847-23be1747eb3a.png)

**Comment by lululxvi** [comment]
Use `PointCloud` https://deepxde.readthedocs.io/en/latest/modules/deepxde.geometry.html#deepxde.geometry.pointcloud.PointCloud
